### PR TITLE
Fix HTML escaping in question tree

### PR DESCRIPTION
**Pull request by millerdev**
http://manage.dimagi.com/default.asp?236328

@orangejenny

### CODE REVIEW

File: src/javaRosa/util.js [hunk 1]
```diff
@@ -281,6 +281,11 @@ define([
                     return $this[0].outerHTML;
                 });
                 text = xml.normalize(text.html());
+                if (escape) {
+                    text = text.replace(/(<)|>/g, function (match, lt) {
+                        return lt ? "&lt;" : "&gt;";
+                    });
```
Comment (millerdev) [inline src/javaRosa/util.js]: `&` chars have already been converted to `&amp;` above on [line 270](https://github.com/dimagi/Vellum/pull/761/files#diff-16525243359147bd8661fbb97da09528R270)

Comment (orangejenny) [inline src/javaRosa/util.js]: I like doing clever things with regexes, but also think that doing two plain string replacements would be slightly more understandable.

**Comment by orangejenny** [comment]
Thank you for grabbing this. Escaping-related bugs are rotten.

**Comment by millerdev** [comment]
✅

File: src/javaRosa/util.js (hunk 1 above)
Comment (millerdev) [inline src/javaRosa/util.js]: Replacing multiple occurrences of a string in JS requires a regexp (with the `g` flag).

Comment (orangejenny) [inline src/javaRosa/util.js]: aha, makes sense

**Pull request merged**